code/modules/surgery/organs/internal/cyberimp/augments_internal.dm

```diff
@@ -142,6 +156,7 @@
 
 /obj/item/organ/internal/cyberimp/brain/anti_stun/proc/reboot()
 	organ_flags &= ~ORGAN_FAILING
+	to_chat(owner, span_purple("Your rebooter implant is ready"))
```

Comment:
This can (reasonably) trigger when there is no owner, so should sanity check that. Also punctuation. Though this could be a balloon alert. 
```suggestion
	if(owner)
		to_chat(owner, span_purple("Your rebooter implant is ready."))
```